flags.go

```diff
@@ -262,6 +262,12 @@ func newApp() (app *cli.App) {
 				Usage: "The number of maximum idle connections allowed per server",
 			},
 
+			cli.BoolFlag{
```

Comment:
You also need to change [here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/1a2d9d6702fc82d19ac13f1c82dbe26094a86b5a/tools/mount_gcsfuse/main.go#L83) to support this flag while mounting through /etc/fs.

Reply:
Added!